XRTK-Core/Packages/com.xrtk.core/Utilities/Solvers/SolverHandler.cs

```diff
@@ -224,10 +224,10 @@ protected virtual void AttachToNewTrackedObject()
                     Handedness = Handedness.Right;
                     break;
                 case TrackedObjectType.Body:
-                    TrackTransform(MixedRealityToolkit.CameraSystem.CameraRig.BodyTransform);
+                    TrackTransform(MixedRealityToolkit.CameraSystem.MainCameraRig.BodyTransform);
```

Comment:
Not much we can do here. I think solvers should require the usage of the camera system.